src/main/java/org/syncninja/util/TwoFilesComparison.java

```diff
@@ -0,0 +1,43 @@
+package org.syncninja.util;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class TwoFilesComparison {
+
+    public static ArrayList<String[]> compareFiles(String newFilePath, String oldFilePath) throws IOException {
```

Comment:
First please don't use a Normal array like String[], use List<List<String> > or create a class that holds the lists for you.